src/EventSubscriber/OgEventSubscriber.php

```diff
@@ -58,6 +58,13 @@ public static function getSubscribedEvents() {
    *   The OG permission event.
    */
   public function provideDefaultOgPermissions(PermissionEventInterface $event) {
+    // @todo This currently sets both the group level and group content level
```

Comment:
Yes this is already reworked in #240. In that issue two separate calls are done to `$event->setPermissions()`: one to set the group-level permissions and one for the generic entity operation permissions.

In a future PR this will get a third call even: to provide the entity operation permissions for the core entity types like node, block etc.